specs/validator/0_beacon-chain-validator.md

```diff
@@ -217,11 +217,7 @@ def get_epoch_signature(state: BeaconState, block: BeaconBlock, privkey: int) ->
     return bls_sign(
```

Comment:
Now you can shrink `bls_sign` into `bls_sign(privkey, hash_tree_root(slot_to_epoch(block.slot)), domain)`

Reply:
It's a smaller line count, but I don't think it brings improved clarity.